src/python/test/test_dxclient.py

```diff
@@ -6268,6 +6268,29 @@ def test_create_user_account_and_set_bill_to_negative(self):
             with self.assertRaisesRegex(subprocess.CalledProcessError,
                                          "DXCLIError"):
                 run(" ".join([cmd, invalid_opts]))
+    
+    def test_create_user_on_behalf_of(self):
```

Comment:
Can we separate this test into  test_create_user_on_behalf_of_negative and  test_create_user_on_behalf_of?